tests/cxx/quoted_includes_first.cc

```diff
@@ -0,0 +1,39 @@
+//===--- quoted_includes_first.cc - test input file for iwyu --------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// Tests that IWYU will respect the --quoted_includes_first option.
+
+#include "tests/cxx/pch.h"  // this is the precompiled header
+#include <iostream>
+#include <list>
+#include <map>
+#include <memory>
+#include "subfolder/indirect_subfolder.h"
+#include "quoted_includes_first.h"
+
+std::unique_ptr<IndirectSubfolderClass> quoted_includes_class::CreateIndirectSubfolderClass() {
+    return std::make_unique<IndirectSubfolderClass>();
```

Comment:
Only two spaces indentation

Reply:
Oh, hmm... `make_unique` is C++14. I think this is now Clang's default, but it could be that there are platforms where this is not true (the default has been different for different targets before). Could you dumb this down to `return std::unique_ptr<IndirectSubfolderClass>(new IndirectSubfolderClass);`? Thanks!

Reply:
Done and done.